molgenis/molgenis-emx2
# fix: pass filter props in directory app

fixes: #3087

how to test:
1) Set filters
2) Click on a biobank
3) Click on "back to catalogue"
4) Filters should now still be set.

todo:
- [ ] try: maybe store filterstate in local storage, having the filterstate bookmarkable is not important.
- [ ] decide: do i also want to fix the browser based back button?
- [ ]  Check if there are other buttons that need to pass the filter state
- [ ] updated testplan to include a test for this fix, including ref to bug using # notation
